src/OpenTK.GLControl/GLControl.cs

```diff
@@ -181,46 +186,46 @@ protected override CreateParams CreateParams
         /// <param name="e">Not used.</param>
         protected override void OnHandleCreated(EventArgs e)
         {
-            if (context != null)
-            {
-                context.Dispose();
-            }
-
-            if (implementation != null)
-            {
-                implementation.WindowInfo.Dispose();
-            }
-
-            if (design_mode)
-            {
-                implementation = new DummyGLControl();
-            }
-            else
-            {
-                implementation = new GLControlFactory().CreateGLControl(format, this);
-            }
+            if (!(m_implementation is DummyGLControl)) { // No need to recreate our DummyGLControl
```

Comment:
There's a lot of brace changes. I believe they go on the next line.

Reply:
Yes, we use the Allman style.

Reply:
Morning Guys,

Sorry for my late reply, I had other stuff going on that needed my immediate attention.
I made changes according to your comments and I hope all is good now. Please don't hesitate if you need more changes. 

Regards Ronald